src/test/jetpack-editor-setup.js

```diff
@@ -117,6 +150,11 @@ describe( 'Jetpack blocks', () => {
 			// Embed variations can use the register block type WP hook to reactivate
 			// already registered variations, so we need to remove all hooks after the tests.
 			removeAllFilters( 'blocks.registerBlockType' );
+
+			// Clean up registered blocks
+			getBlockTypes().forEach( ( block ) => {
+				unregisterBlockType( block.name );
+			} );
```

Comment:
This code block comes from [here](https://github.com/wordpress-mobile/gutenberg-mobile/pull/5293/files#diff-387560ba85f15f30d57e115a8bb657f8163c2e53fbc02a1e3e9b467779dc50f4L56-L58) as for embed variations we still need to unregister the block types due to the fact of registering all core blocks.